.circleci/config.yml

```diff
@@ -0,0 +1,75 @@
+version: 2.1
+
+jobs:
+
+  build:
+    docker:
+      - image: cimg/base:2022.01
+    steps:
+      - checkout
+      - run:
+          name: What does the directory layout look like?
+          command: |
+            pwd
+            ls -lh .
+
+      - run:
+          name: Add OS packages
+          command: |
+            set -euo pipefail
+            sudo apt-get update
+            sudo apt-get install -y php7.4-dev libuv1-dev
+
+      - run:
+          name: Add /opt packages
```

Comment:
circle CI has a pretty cool caching mechanism, we could use concatenation of both keys as a cache for the job and indicate /opt as result of the cache.

Reply:
Great idea. Maybe we can cache in another PR.